common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -408,44 +402,24 @@ def check_button_name(self):
 
         Usually it is just "Check", but if this is the student's
         final attempt, change the name to "Final Check".
-        The text can be customized by the text_customization setting.
         """
         # The logic flow is a little odd so that _('xxx') strings can be found for
         # translation while also running _() just once for each string.
         _ = self.runtime.service(self, "i18n").ugettext
-        check = _('Check')
-        final_check = _('Final Check')
-
-        # Apply customizations if present
-        if 'custom_check' in self.text_customization:
-            check = _(self.text_customization.get('custom_check'))                # pylint: disable=translation-of-non-string
-        if 'custom_final_check' in self.text_customization:
-            final_check = _(self.text_customization.get('custom_final_check'))    # pylint: disable=translation-of-non-string
-        # TODO: need a way to get the customized words into the list of
-        # words to be translated
-
-        if self.max_attempts is not None and self.attempts >= self.max_attempts - 1:
-            return final_check
-        else:
-            return check
+        check = _('Submit')
+
+        return check
 
     def check_button_checking_name(self):
```

Comment:
I put some notes about cleanup like this in https://openedx.atlassian.net/browse/TNL-4939.